### PR TITLE
Add claim rewards

### PR DESCRIPTION
**Pull request by trentkrogers**
### 📝 Description

Adds Osmosis claim rewards functionality

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

https://user-images.githubusercontent.com/16307439/183135175-7deae030-5f0a-4693-9995-0c6191838337.mp4

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/families/osmosis/index.js [hunk 1]
```diff
@@ -2,5 +2,6 @@
 import * as OsmosisDelegationFlow from "./DelegationFlow";
 import * as OsmosisUndelegationFlow from "./UndelegationFlow";
 import * as OsmosisRedelegationFlow from "./RedelegationFlow";
+import * as OsmosisClaimRewardsFlow from "./ClaimRewardsFlow";
 
-export { OsmosisDelegationFlow, OsmosisRedelegationFlow , OsmosisUndelegationFlow };
\ No newline at end of file
+export { OsmosisDelegationFlow, OsmosisRedelegationFlow , OsmosisUndelegationFlow, OsmosisClaimRewardsFlow };
```
Comment (JesseKuntz) [inline apps/ledger-live-mobile/src/families/osmosis/index.js]: I think eslint complains here, and maybe add a newline?

**Review by JesseKuntz — COMMENTED** [review]
Great work! Minor comment

**Review by JesseKuntz — APPROVED** [review]
LGTM 🔥

File: apps/ledger-live-mobile/src/families/osmosis/ClaimRewardsFlow/01-SelectValidator.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
+// @flow
+import invariant from "invariant";
+import React, { useCallback } from "react";
+import { View, StyleSheet, FlatList } from "react-native";
+import { useSelector } from "react-redux";
+
+import type { Transaction } from "@ledgerhq/live-common/families/cosmos/types";
```
Comment (marco-figment) [inline apps/ledger-live-mobile/src/families/osmosis/ClaimRewardsFlow/01-SelectValidator.js]: this should be
`import type { Transaction } from "@ledgerhq/live-common/families/osmosis/types";`

File: apps/ledger-live-mobile/src/families/osmosis/ClaimRewardsFlow/02-SelectMethod.js [hunk 3]
```diff
@@ -0,0 +1,325 @@
+// @flow
+import invariant from "invariant";
+import React, { useCallback, useState } from "react";
+import { View, StyleSheet, TouchableOpacity } from "react-native";
+import SafeAreaView from "react-native-safe-area-view";
+import { Trans } from "react-i18next";
+import { useSelector } from "react-redux";
+import { BigNumber } from "bignumber.js";
+
+import type {
```
Comment (marco-figment) [inline apps/ledger-live-mobile/src/families/osmosis/ClaimRewardsFlow/02-SelectMethod.js]: same here: 
```suggestion
import type { CosmosValidatorItem } from "@ledgerhq/live-common/families/cosmos/types"
import type { Transaction } from "@ledgerhq/live-common/families/osmosis/types"
```

**Review by marco-figment — CHANGES_REQUESTED** [review]
nice job! could you please fix those couple of imports?

File: apps/ledger-live-mobile/src/families/osmosis/ClaimRewardsFlow/01-SelectValidator.js (hunk 2 above)
Comment (trentkrogers) [reply apps/ledger-live-mobile/src/families/osmosis/ClaimRewardsFlow/01-SelectValidator.js]: Got it. Should this also be updated in the Redelegation flow? @JesseKuntz

**Review by marco-figment — APPROVED** [review]
spongebob_thumbs_up

File: apps/ledger-live-mobile/src/families/osmosis/ClaimRewardsFlow/01-SelectValidator.js (hunk 2 above)
Comment (marco-figment) [reply apps/ledger-live-mobile/src/families/osmosis/ClaimRewardsFlow/01-SelectValidator.js]: done.

**Pull request merged**